odkx-src/xlsx-converter-reference.rst

```diff
@@ -395,8 +433,11 @@ The following prompt types are available in ODK-X Survey.
   * - video
     - | Used to capture a video.
   * - textarea
-    - | Used to enter the information in a big text area or paragraphs.
-    
+    - | Used to enter the information in a big text area or paragraphs. 
+
+.. note::
+  The Non-Gregorian dates are saved to the database in a converted Gregorian date time but is displayed to the user as a Gregorian date.
```

Comment:
Is it displayed as Gregorian or non-Gregorian?

Reply:
Non-gregorian, actually. I'll correct it.

Reply:
Great, thanks for fixing.